includes/classes/Indexable/User/User.php

```diff
@@ -737,8 +737,17 @@ public function query_db( $args ) {
 		/**
 		 * WP_User_Query doesn't let us get users across all blogs easily. This is the best
 		 * way to do that.
+		 *
+		 * The $wpdb->prepare will quate placeholders.
+		 *
+		 * SELECT * FROM table 'ORDER BY col asc'; - mariadb (vipd and local dev-env) doesn't like that
+		 * SELECT * FROM table ORDER BY 'col' 'asc'; - (this is the upstream variant) Vitess (k8s) doesn't like that
+		 *
+		 * We are sanitizing orderby in advance and putting it as a varible to avoid quates.
 		 */
-		$objects = $wpdb->get_results( $wpdb->prepare( "SELECT SQL_CALC_FOUND_ROWS ID FROM {$wpdb->users} %s LIMIT %d, %d", $orderby, (int) $args['offset'], (int) $args['number'] ) );
+		// @codingStandardsIgnoreStart
```

Comment:
Could use input on this one. But with the sanitize_order_by above I think this is safe.

Reply:
Seems pretty strict on what is allowed. The only alternative would be to provide an allowlist of valid `$args['orderby']` columns and ensure it's one of them - but I don't feel like that is necessary here as it's no a top-level user API really.